src/libMessage/ZilliqaMessage.proto

```diff
@@ -78,8 +78,9 @@ message ProtoDSBlock
             optional ByteArray val      = 2;  // Added in: v1.0, Deprecated in: N/A
             // Add new members here
         }
-        repeated PowDSWinners dswinners = 10;  // Added in: v1.0, Deprecated in: N/A
+        repeated PowDSWinners dswinners = 10; // Added in: v1.0, Deprecated in: N/A
         optional DSBlockHashSet hash    = 11; // Added in: v1.0, Deprecated in: N/A
+        repeated ByteArray dsremoved = 12; // Added in: v4.5, Deprecated in: N/A
```

Comment:
Change to `added in: v2.0` since the version is from block version

Reply:
Thanks, made the change in https://github.com/Zilliqa/Zilliqa/pull/1587/commits/5f896af587a3653bb5cc0aac0d518253d0ba6819